homeworks/12/chat-system/src/Client.java

```diff
@@ -0,0 +1,99 @@
+import java.io.*;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Client {
+    private Socket socket;
+    private BufferedReader bufferedReader;
+    private BufferedWriter bufferedWriter;
+    private String nickname;
+
+    public Client(Socket socket, String nickname) {
+        try {
+            this.socket = socket;
+            bufferedReader = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            bufferedWriter = new BufferedWriter(new OutputStreamWriter(socket.getOutputStream()));
+            this.nickname = nickname;
+        } catch (IOException exception){
+            try {
+                bufferedReader.close();
+                bufferedWriter.close();
+                socket.close();
+            }catch (Exception exception1) {
+                System.out.println(exception1);
+            }
+        }
+    }
+
+    public void sendMessage() {
+        try {
+            bufferedWriter.write(nickname);
+            bufferedWriter.newLine();
+            bufferedWriter.flush();
+
+            Scanner scanner = new Scanner(System.in);
+            while (socket.isConnected()) {
+                String message = scanner.nextLine();
+                if(message.startsWith("/msg")) {
+                    String newString = message.substring("/msg".length());
+                    bufferedWriter.write(nickname + ": " + newString);

```

Comment:
You have not added timestamp before messges like in the example:

`[10:33 AM] Pesho: zdr`